code-challenges.js

```diff
@@ -10,30 +10,81 @@
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp = 42
-// var temp = 350
-// var temp = 212
-
+var temp = 350
+var temp = 212
+
+//***********************//
+var tempA = 42
+var tempB = 350
+var tempC = 212
+var tempS = [tempA, tempB, tempC]
+    for(i=0; i < tempS.length; i++) {
+      if(temp > 212){
+        console.log(`${temp[i]} is above boiling point`);
+      } else if (temp === 212){
+        console.log(`${temp[i]} is at boiling point`);
+      } else {
+        console.log(`${temp[i]} is below boiling point`);
+      }
+    }
```

Comment:
Your logic looks great, but the goal of these challenges is to work from the exact given input and produce the exact expected output.